File: from-3101-to-3200/3192-minimum-operations-to-make-binary-array-elements-equal-to-one-ii/solution.py
class Solution:
    def minOperations(self, nums: List[int]) -> int:
        flipped = False
        result = 0
        for num in nums:
            if num == 0 and not flipped or num == 1 and flipped:
                result += 1
                flipped = not flipped
        return result

File: from-3101-to-3200/3192-minimum-operations-to-make-binary-array-elements-equal-to-one-ii/solution.go
func minOperations(nums []int) int {
    result := 0
    flipped := false
    for _, num := range nums {
        if num == 0 && !flipped || num == 1 && flipped {
            result += 1
            flipped = !flipped
        }
    }
    return result
}